src/raftstore/store/fsm/peer.rs

```diff
@@ -1446,7 +1465,11 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         }
         {
             let mut meta = self.ctx.store_meta.lock().unwrap();
-            meta.set_region(&self.ctx.coprocessor_host, cp.region, &mut self.fsm.peer);
+            meta.set_region(
+                &self.ctx.coprocessor_host,
+                cp.region.clone(),
```

Comment:
Is it required?

Reply:
Otherwise, `cp` is moved by `set_region`.